### ISSUE TITLE
ffe-chart-donut-react should have a title in the generated SVG (a11y)

### ISSUE THREAD

**Issue by ivarni** [SpareBank1/designsystem#559]
#### Which package is this issue related to?

ffe-chart-donut-react

#### Describe your issue (screenshots welcome!)

This package generates a SVG pie-chart that doesn't include a `<title>` child, making it less accessible for screen-readers.

[css-tricks](https://css-tricks.com/accessible-svgs/#article-header-id-6) recommends adding a title tag that describes the contents of the SVG. It should be possible to solve this in a generic way using the `name`, `firstLabel` and `lastLabel` props, but it could also take a non-required prop function to give the consumer more control over the title.

**Comment by 42tte** [comment]
No need for redundant information for screen readers.
If there is a use case where the SVGs are more than a visual presentation of `name`, `firstLabel` and `secondLabel` and `<title>` is not only a copy of the same - then there is/might be a reason to include it.

**Comment by ivarni** [comment]
This is specific to what this module produces, not images in general

![image](https://user-images.githubusercontent.com/379849/53001093-2aa15e00-342a-11e9-83cf-dc5f0d117be6.png)

There's not much to go on here for a screenreader as the percentages in the piechart are embedded deeply in the SVGs markup so I would argue that there's absolutely a reason to include that information (in this specific example, *53% aksjefond, 47% rentefond*) in a `<title>` tag.

**Comment by 42tte** [comment]
As far as I can see from [ChartDonut](https://design.sparebank1.no/styleguidist/index.html#!/ChartDonut) the text is accessible. However you are right that it could be better catered in one sentence than split in five.

![screenshot 2019-02-19 at 10 27 29](https://user-images.githubusercontent.com/8436510/53004367-5247f480-3431-11e9-98e7-c6d54a17c702.png)

**Comment by ivarni** [comment]
I'm guessing the screenreader don't really care if it's split into several `<div>`s or not any more than it would care if we used elements to apply different styling to words in a sentence so I suppose we're good then.
